package/MDAnalysis/analysis/diffusionmap.py

```diff
@@ -269,6 +269,7 @@ def _single_frame(self):
             self.results.dist_matrix[self._frame_index,
                                      j+self._frame_index] = (
                                             dist if dist > self._cutoff else 0)
+            # TODO: move this calculation to _conclude
```

Comment:
Can you please give a bit more context to this TODO (i.e. it is something that still needs to be tackled in this PR or is it something for the future)? If it's not relevant, please remove.